### PR TITLE
Small number of bug fixes.

### PR DESCRIPTION
**Pull request by jorive**
- Added stability prefix to the scenario benchmark (JitBench)
- Specify output directory to the `run-xunit-perf.cmd` script and avoid the extra step to xcopy files to the archive folder.
- Added a command line parser class to the illink scenario, and changed its behavior where it used to fail when a new command line option passed to xUnit was not recognized.

### CODE REVIEW

**Comment by jorive** [comment]
@dotnet-bot test ci please

File: tests/scripts/run-xunit-perf.cmd [hunk 1]
```diff
@@ -96,21 +96,40 @@ setlocal
   rem CORE_ROOT environment variable is used by some benchmarks such as Roslyn / CscBench.
   set CORE_ROOT=%LV_SANDBOX_DIR%
   set LV_RUNID=Perf-%ETW_COLLECTION%
-  set BENCHNAME_LOG_FILE_NAME=%LV_BENCHMARKS_OUTPUT_DIR%\%LV_RUNID%-%BENCHNAME%.log
 
+  set "LV_BENCHMARK_OUTPUT_DIR="
```
Comment (DrewScoggins) [inline tests/scripts/run-xunit-perf.cmd]: I'm confused by this part.  Why do we set this to something on line 13 above, and then unset it here?

File: tests/src/performance/linkbench/linkbench.cs [hunk 2]
```diff
@@ -272,107 +272,41 @@ public class LinkBench
...
-        }
+            var options = BenchmarkOptions.Parse(args);
 
-        public static int Main(String [] args)
-        {
-            bool doSetup = true;
-            bool doBuild = true;
-            string runId = "";
-            string outputdir = ".";
-            string runOne = null;
```
Comment (DrewScoggins) [inline tests/src/performance/linkbench/linkbench.cs]: Was this variable never used? You remove it here, but it is no where else in the diff

**Comment by DrewScoggins** [comment]
Can we also update this line, https://github.com/jorive/coreclr/blob/cd4d72523bfc865003be81d1d49b86ed551cb8df/tests/scripts/run-xunit-perf.cmd#L323, so that we can save off the output from the tests themselves. This would make it much easier to diagnose issues.

File: tests/scripts/run-xunit-perf.cmd (hunk 1 above)
Comment (adiaaida) [reply tests/scripts/run-xunit-perf.cmd]: Agreed this seems weird.

Comment (adiaaida) [reply tests/scripts/run-xunit-perf.cmd]: Oh, it's a different variable. LV_SANDBOX on line 13, LV_BENCHMARK here.

**Comment by jorive** [comment]
Good call. I will add it. Thanks!

---
In reply to: [342590300](https://github.com/dotnet/coreclr/pull/14900#issuecomment-342590300) [](ancestors = 342590300)

File: tests/src/performance/linkbench/linkbench.cs (hunk 2 above)
Comment (jorive) [reply tests/src/performance/linkbench/linkbench.cs]: Yes, it was never used (maybe I should make the projects treat warnings as errors).

---
In reply to: [149472900](https://github.com/dotnet/coreclr/pull/14900#discussion_r149472900) [](ancestors = 149472900)

**Comment by jorive** [comment]
@adiaaida What is the ETA for having a single "perf" groovy file?

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
@adiaaida Ported changes to pipeline job

File: buildpipeline/perf-pipeline.groovy [hunk 3]
```diff
@@ -334,7 +339,7 @@ if (!isPR()) {
     }
 
     outerLoopTests["windows ${arch} ryujit full_opt pgo${baseline} illink"] = {
-        simpleNode('windows_server_2015_clr_perf', 180) {
+        simpleNode('windows_server_2015_clr_perf', 180) { // This label does not exist in Jenkins!
```
Comment (adiaaida) [inline buildpipeline/perf-pipeline.groovy]: Could you update the label with the correct one?

**Comment by DrewScoggins** [comment]
LGTM

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by adiaaida** [comment]
LGTM

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
There has been network issues in the lab that has stopped the perf runs to complete successfully. If everybody is OK with my changes, I can go ahead and merge.

**Comment by adiaaida** [comment]
Sounds good to me

**Pull request merged**